tests/labhub_test.py

```diff
@@ -72,6 +71,9 @@ def test_hello_world_callback(self):
         self.mock_team.invite.assert_called_with(None)
 
     def test_create_issue_cmd(self):
+        plugins.labhub.GitHub = create_autospec(IGitt.GitHub.GitHub.GitHub)
+        plugins.labhub.GitLab = create_autospec(IGitt.GitLab.GitLab.GitLab)
```

Comment:
not unrelated, I don't want these in the test_mark_cmd, hence moved from setUp to tests where they were required